### ISSUE TITLE
Bug: [TUI clearScreen function not working in Ubuntu]

### ISSUE THREAD

**Issue by slraina** [AndrewRoe34/agile-planner#53]
**Describe the bug**
The cls command in TUIState interface doesn't work in ubuntu. Hence Gradle is unable to build code 

**To Reproduce**
1. Go to Ubuntu Desktop OS.
2. Clone the code from Github.
3. Replicate the steps mentioned in Readme to create build. 
4. See error as it won't be able to run as ubuntu doesn't support cls command listed under TUIState Interface's clearScreen() method.

**Expected behavior**
The code was supposed to build and run successfully leading to UI for console application of Agile Planner.

**Desktop (please complete the following information):**
 - OS: Ubuntu 23.10

**Comment by AndrewRoe34** [comment]
This has been known for some time. You would need to update the clearScreen() static method to check the OS and see if it's Windows or Linux to fix the issue.

For now, you can simply comment out the actual code like follows to run it:
```java
package com.planner.ui;

public interface TUIState {

    void setupAndDisplayPage();

    static void clearScreen() {
        /*try {
            new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor();
        } catch (Exception e) {
            throw new RuntimeException(e);
        }*/
    }
}
```

**Comment by AndrewRoe34** [comment]
Or, you can replace the commented out code with a call to Linux's `clear` if you would like to see how it operates under normal conditions.

**Comment by slraina** [comment]
> Or, you can replace the commented out code with a call to Linux's `clear` if you would like to see how it operates under normal conditions.

Yes, but i was thinking of creating a simple check of getting the OS name and assigning it to a variable if it happens to be windows we'll proceed as usual and in case it is linux/unix based then we'll execute the `clear` command.

I have done same in my branch should i proceed with a PR @AndrewRoe34  ?

**Comment by AndrewRoe34** [comment]
> > Or, you can replace the commented out code with a call to Linux's `clear` if you would like to see how it operates under normal conditions.
> 
> 
> 
> Yes, but i was thinking of creating a simple check of getting the OS name and assigning it to a variable if it happens to be windows we'll proceed as usual and in case it is linux/unix based then we'll execute the `clear` command.
> 
> 
> 
> I have done same in my branch should i proceed with a PR @AndrewRoe34  ?

Absolutely! I'll review changes and make sure it runs both via Windows and Linux :)

**Comment by slraina** [comment]
Hey @AndrewRoe34 , not able to push my PR, please provide access for the same !

**Comment by AndrewRoe34** [comment]
@slraina Just added you to the repo (just approve request). Surprised that you weren't able to create a PR.

**Comment by AndrewRoe34** [comment]
Issue was fixed with [PR#54](https://github.com/AndrewRoe34/agile-planner/pull/54)
